src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -239,6 +239,21 @@ private function addFormSection(ArrayNodeDefinition $rootNode, callable $enableI
                         ->end()
                         // to be deprecated in Symfony 6.1
                         ->booleanNode('legacy_error_messages')->end()
+                        // to be deprecated in Symfony 7.1
+                        ->booleanNode('controllers_render_form_views')
+                            ->info('Enables "AbstractController::render()" method to convert FormInterface to FormView and setting 422 status code when invalid.')
+                            ->defaultFalse()
+                            ->validate()
+                                ->always()
+                                ->then(function ($v) {
+                                    if (false === $v) {
+                                        trigger_deprecation('symfony/framework-bundle', '6.2', 'Setting the "framework.form.controllers_render_form_views" option to "false" is deprecated. It will have no effect as of Symfony 7.0.');
```

Comment:
instead of using `->always()` and then doing the special logic only when `false`, use `->ifFalse()` instead

Reply:
That was my first attempt before doing this, but here's what I get when I try:
```
Error : Call to undefined method Symfony\Component\Config\Definition\Builder\ExprBuilder::ifFalse()
```
I could have used `ifEmpty()` but I preferred to be strict. WDYT, should I change this or should we add `ifFalse()` to the expression builder?